src/rpdk/core/contract/suite/handler_read.py

```diff
@@ -10,5 +10,7 @@
 
 @pytest.mark.read
 def contract_read_without_create(resource_client):
-    model = resource_client.generate_create_example()
+    model = (
+        resource_client.generate_invalid_create_example()
```

Comment:
don't understand the comment here. is this related to the writable id skip?

Reply:
not really it's no longer id skip. it's more like to use invalid primary id to assert read handler throws not found